### PR TITLE
Add custom fontsize support for notifications

### LINKED ISSUE #4866: Increase hyprland notification font size
### Description

I'd love it if it were possible to increase the font size of the notifications created via the `hyprctl notify` command.

### PR DESCRIPTION
**Pull request by ItsDrike**
#### Describe your PR, what does it fix/add?

This adds support for specifying a custom font size the notification will be rendered with.

Fixes #4866

image

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)

This is technically a breaking change for anyone who is currently using `hyprctl notify`, as it requires an extra argument for the font size.

Note that I'm not entirely sure how the font size calculation worked in the original code here:

```cpp
        const auto            FONTSIZE        = std::clamp((int)(notif->fontSize * ((pMonitor->vecPixelSize.x * SCALE) / 1920.f)), 8, 40);
```

I pretty much just reused this code and used `notif->fontSize`, instead of the hard-coded value. I think this is the correct way to do it, but I'm not certain of it.

#### Is it ready for merging, or does it need work?

Ready.

### CODE REVIEW

**Review by vaxerski — COMMENTED** [review]
this is breaking, though. would be better not to be

**Comment by ItsDrike** [comment]
Yeah, I agree, but I'm not sure how to achieve that when all of the arguments after 5th were just treated as parts of the msg. Any suggestions?

**Comment by vaxerski** [comment]
why not make this just a config variable

**Comment by ItsDrike** [comment]
I considered that, but I figured the issue author wanted to be able to set different sizes for different notifs. If a single size is enough though, I can modify this

**Comment by vaxerski** [comment]
I am for specifying args at the beginning of the message then, like:
`fontsize:17 Hello Hyprland!`

File: src/debug/HyprCtl.cpp [hunk 1]
```diff
@@ -1420,17 +1420,33 @@ std::string dispatchNotify(eHyprCtlOutputFormat format, std::string request) {
...
+        } catch (std::exception& e) { return "invalid fontsize karg"; }
+
+        ++msgidx;
     }
 
+    if (vars.size() <= msgidx)
+        return "not enough args";
+
+    std::string message = "";
+    for (size_t i = msgidx; i < vars.size(); ++i) {
```
Comment (vaxerski) [inline src/debug/HyprCtl.cpp:1444]: `.join` exists.

Comment (ItsDrike) [reply src/debug/HyprCtl.cpp]: k fixed (though I was just reusing your code there)

**Pull request merged**